src/connectors/nugs.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+Connector.playerSelector = 'div[app-player] .player-wrapper';
+
+Connector.playButtonSelector = '.play-toggle';
```

Comment:
Actually, spoke too soon here.

Here, you should avoid using both playbuttonselector and pausebuttonselector. Please remove one of the two.

As soon as that's done this can be merged I think.

Reply:
Ah, I wondered about that. Is one preferred over the other? It's the same button but when playing the pause class toggles, but the okay class remains all the time.

Reply:
As far as I am aware, there is no preference of one to the other.
The reason both exist is in case only one of the two can be used on a site, using both at once is redundant.
Just verify that it still works with the one you stick with and it should be good.

Reply:
I first removed the pause, but after testing realized it wasn't picking up anymore, so swapped it back for the pause button and I'm seeing the scrobbled. I'll consider this resolved.